src/ui/Analytics/AnalyticsClient.ts

```diff
@@ -91,8 +91,9 @@ export interface IAnalyticsClient {
    * @param result The result that was clicked.
    * @param element The HTMLElement that the user has clicked in the interface. Default value is the element on which
    * the `Analytics` component is bound.
+   * @returns {Promise<Any>|Promise<IAPIAnalyticsEventResponse>|Promise<IAPIAnalyticsEventResponse[]>}
```

Comment:
Since those interface(s) are not documented publicly, there's not much point in saying it returns those types.

Let's keep it simple, just remove the `@returns`.